vmdb/app/models/orchestration_stack.rb

```diff
@@ -18,6 +18,10 @@ class OrchestrationStack < ActiveRecord::Base
   has_many   :outputs,    :dependent => :destroy, :foreign_key => :stack_id, :class_name => "OrchestrationStackOutput"
   has_many   :resources,  :dependent => :destroy, :foreign_key => :stack_id, :class_name => "OrchestrationStackResource"
 
+  alias stack_parameters parameters
+  alias stack_outputs outputs
+  alias stack_resources resources
```

Comment:
@Fryguy @chessbyte This aliases are needed because of the reporting. At the first attempt I added `resources` to the `@@include_tables` in `MiqExpression`. It caused a lot of tests failed. The aliases help to uniquely identify these attributes only comes from stacks.